template/src/utils/trimText/trimText.test.ts

```diff
@@ -0,0 +1,17 @@
+import { trimText } from './trimText';
+
+describe('trimText', () => {
+  test('should trim a long string to the alloted length', () => {
```

Comment:
nit: how do you feel about using `it()` instead of `test()` ? It's the same function, just reads more like a sentence. I have a strong preference for it as a standard, coming from the BDD habits I developed from Ruby on Rails and all that. LMK what you think though.

Reply:
I'm definitely more used to `it` and prefer it as well. I will update in a follow-up branch I'm doing. Thanks!